internal/configs/oidc/oidc.conf

```diff
@@ -15,6 +15,7 @@
         proxy_method GET;                             # In case client request was non-GET
         proxy_set_header Content-Length "";           # ''
         proxy_pass $oidc_jwt_keyfile;                 # Expecting to find a URI here
+        proxy_set_header    Accept-Encoding "gzip";
```

Comment:
From my understanding this has the downstream impact of setting this as the default for all customers instead of implement this as an option that can be enabled or disabled as needed by any customer.